fiscalyear.py

```diff
@@ -53,8 +100,12 @@ def _validate_fiscal_calendar_params(start_year, start_month, start_day):
     _check_day(start_month, start_day)
 
 
-def setup_fiscal_calendar(start_year=None, start_month=None, start_day=None):
-    """Modify the start of the fiscal calendar.
+def setup_fiscal_calendar(start_year=None, start_month=None, start_day=None,
+                          end_month=None, end_day=None):
+    """Modify the fiscal calendar.
+
+    If either of ``start_month`` or ``start_day`` are present, ``end_month``
+    and ``end_day`` are ignored.
```

Comment:
This might be surprising behavior. I think it would be preferable to throw an error when invalid combinations of arguments are supplied